conceptnet5/nodes.py

```diff
@@ -106,6 +108,8 @@ def standardized_concept_uri(lang, text, *more):
     '/c/en/this_is_test'
     >>> standardized_concept_uri('en', 'this is a test', 'n', 'example phrase')
     '/c/en/this_is_test/n/example_phrase'
+    >>> standardized_concept_uri('sh', 'симетрија')
```

Comment:
The code looks good. One change I'd make is adding the description of this PR to the docstring for `standardized_concept_uri`.